### PR TITLE
virtcontainers: Add to bridges unit test

### LINKED ISSUE #2827: Increase unit test coverage for virtcontainers
There are gaps in unit test coverage for virtcontainers/types/bridges. Expand unit test to include coverage for `NewBridge` and `AddressFormat...` functions.

### PR DESCRIPTION
**Pull request by cmaf**
Add function that creates new bridges to increase unit test coverage
for virtcontainers/types/bridges.

Fixes #2827

Signed-off-by: Chelsea Mafrica <chelsea.e.mafrica@intel.com>

### CODE REVIEW

**Comment by cmaf** [comment]
I am still adding to this for more coverage for this unit test but a review is appreciated.

File: virtcontainers/types/bridges_test.go [hunk 1]
```diff
@@ -43,6 +43,40 @@ func testAddRemoveDevice(t *testing.T, b *Bridge) {
...
+
+func TestNewBridge(t *testing.T) {
+	assert := assert.New(t)
+
+	var pci Type = "pci"
+	var pcie Type = "pcie"
+	var ccw Type = "ccw"
+
+	var pcibridge = NewBridge(PCI, "", make(map[uint32]string), 0)
+	assert.Equal(pcibridge.Type, pci, "Type should be PCI")
```
Comment (jcvenegas) [inline virtcontainers/types/bridges_test.go:71]: You could also assert the values of map and addr

File: virtcontainers/types/bridges_test.go [hunk 2]
```diff
@@ -43,6 +43,40 @@ func testAddRemoveDevice(t *testing.T, b *Bridge) {
 	}
 }
 
+func TestAddressFormat(t *testing.T) {
+	assert := assert.New(t)
+
+	var ccwbridge = NewBridge(CCW, "", make(map[uint32]string), 0)
+	_, err := ccwbridge.AddressFormatCCW("0")
```
[the remark this answers is not in the document]
Comment (cmaf) [reply virtcontainers/types/bridges_test.go]: Applied these changes

**Comment by cmaf** [comment]
@jcvenegas I updated the test. I added a test for the default bridge and I think I hit 100% coverage. 

Should I add a test for every parameter? I left `ID` as an empty string but I could test it, and I only test `maxCapacity` for the default bridge.

**Comment by jodh-intel** [comment]
@cmaf - could you port this to the https://github.com/kata-containers/kata-containers/tree/2.0-dev branch too please.

**Comment by cmaf** [comment]
Ported to 2.0 with [423](https://github.com/kata-containers/kata-containers/pull/423)
